test/extended/util/db/mongodb.go

```diff
@@ -0,0 +1,62 @@
+package db
+
+import (
+	"errors"
+	"fmt"
+
+	"github.com/openshift/origin/test/extended/util"
+)
+
+// MongoDB is a MongoDB helper for executing commands.
+type MongoDB struct {
+	PodName       string
+	MasterPodName string
+}
+
+// NewMongoDB queries OpenShift for a pod with given name, saving environment
+// variables like username and password for easier use.
+func NewMongoDB(podName, masterPodName string) util.Database {
+	if masterPodName == "" {
+		masterPodName = podName
+	}
+	return &MongoDB{
+		PodName:       podName,
+		MasterPodName: masterPodName,
+	}
+}
+
+func (m MongoDB) GetPodName() string {
+	return m.PodName
```

Comment:
>why a getter when the field is exported?

Looks like we have a getter because we need to have a way to get this member from different structs:

```go
func WaitUntilUp(oc *CLI, d Database, timeout time.Duration) error {
	...
	if err == wait.ErrWaitTimeout {
		return fmt.Errorf("timed out waiting for pod %s get up", d.GetPodName())
	}
	return err
}
```

Are you still thinking that I should remove it? At least I can un-export it.